.github/workflows/cloud_continuous_integration.yml

```diff
@@ -7,7 +7,7 @@ on:
       - '.github/workflows/cloud_continuous_integration.yml'
       - '.github/workflow_scripts/test_cloud.sh'
   pull_request_target:
-    types: [labeled, synchronize]
+    types: [labeled, synchronize, opened]
```

Comment:
do we need this change for IAM policy refinement?

Reply:
No. It's just a quick fix for the CI. Didn't want to bother to create a specific PR just for this